rest_framework/fields.py

```diff
@@ -1227,6 +1227,8 @@ def to_internal_value(self, value):
         self.fail('invalid', format='[DD] [HH:[MM:]]ss[.uuuuuu]')
 
     def to_representation(self, value):
+        if isinstance(value, six.string_types):
+            return value
```

Comment:
This implies that `DurationField` native type could be string already instead of time delta.
Why is this needed by the YAML renderer ?